charts/nextcloud/templates/service.yaml

```diff
@@ -13,6 +13,13 @@ spec:
   {{- if eq .Values.service.type "LoadBalancer" }}
   loadBalancerIP: {{ default "" .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if and .Values.service.loadBalancerClass (eq .Values.service.type "LoadBalancer") }}
```

Comment:
Can you change the order of the statements so that it matches the one above more closely?

Reply:
I modified the conditions and the scope of the if statement